Cython/Compiler/Symtab.py

```diff
@@ -752,7 +754,8 @@ def register_pyfunction(self, entry):
 
     def declare_cfunction(self, name, type, pos,
                           cname=None, visibility='private', api=0, in_pxd=0,
-                          defining=0, modifiers=(), utility_code=None, overridable=False):
+                          defining=0, modifiers=(), utility_code=None,
+                          overridable=False):
```

Comment:
These are now unfortunate, unnecessary changes. Could you undo the signature reformatting?

Reply:
done